tm2/pkg/std/coin_benchmark_test.go

```diff
@@ -7,6 +7,7 @@ import (
 )
 
 func BenchmarkCoinsAdditionIntersect(b *testing.B) {
+	b.Skip("TODO: panicking benchmark")
```

Comment:
Can we open up an issue for these?

Reply:
https://github.com/gnolang/gno/issues/907